app/javascript/components/Reservations.js

```diff
@@ -4,11 +4,13 @@ import { fetchReservationsApi } from '../redux/features/reservations/reservation
 import { fetchSections } from '../redux/features/sections/sectionsSlice';
 import '../../assets/stylesheets/reservations.css';
 import { CiUser } from "react-icons/ci";
+import NavBar from './NavBar';
 
 function Reservations() {
```

Comment:
- [ ] The reservations page is not scrollable:

![image](https://github.com/JuanDBta/booking-bites/assets/57421823/e71385e4-89ac-442a-8ddb-54cdb272311f)

Therefore, I can't see the whole list of reservations (there should be 7 in this case):

![image](https://github.com/JuanDBta/booking-bites/assets/57421823/7d5ded8a-dee7-43dc-9156-b31d97e81315)

Kindly, please touch up your styling to make sure the whole reservations list can be viewed.  👍